Sources/BraveWallet/WalletStrings.swift

```diff
@@ -64,6 +64,13 @@ extension Strings {
       value: "Assets",
       comment: "The title which is displayed above a list of assets/tokens"
     )
+    public static let nftsTitle = NSLocalizedString(
+      "wallet.nftsTitle",
+      tableName: "BraveWallet",
+      bundle: .module,
+      value: "NFTs",
+      comment: "The title which is displayed above a list of NFTs"
+    )
```

Comment:
Do we even have to translate this? If so, best to mention that NFT is an acronym in the comments for translators

Reply:
Yeah not sure, I'll update comment to mention it's an acronym in case of pluralization changes.

Reply:
Updated in [d83aa86](https://github.com/brave/brave-ios/pull/6380/commits/d83aa86134c9d71543df6dbe78e741aff58e92af)